### ISSUE TITLE
No detections occurs with COCO model

### ISSUE THREAD

**Issue by FamousDirector** [NVIDIA/retinanet-examples#8]
I have trained an model on the COCO dataset according to the example you have in the README.md:
```
retinanet train retinanet_rn50fpn.pth --backbone ResNet50FPN \
    --images /coco/images/train2017/ --annotations /coco/annotations/instances_train2017.json \
    --val-images /coco/images/val2017/ --val-annotations /coco/annotations/instances_val2017.json \
    --logdir /project/output/logs/
```
I let it run for  90000 iterations. However, when it trains it prints a `No detections!` I confirm that it is not working when I run inference on the COCO validation set:
```
retinanet infer retinanet_rn50fpn.pth  --images /coco/images/val2017/ 
```
I am using a single V100 to train and test.

**Comment by pkashinkunti** [comment]
If using the default learning rate with 1 GPU, the loss might diverge (The default value works well for 8 GPUs). Using a lower learning rate would be required in this case but, if too small, the network would not learn at all! It requires some tuning. What learning rate did you use in this experiment?

Also, if you had any training logs that would also be helpful to figure out what's going on.

**Comment by FamousDirector** [comment]
I used the default learning rate. After I ran with a lower `lr` and a custom dataset I had no issue. Closing.

**Comment by VincentDuf** [comment]
Hi,
I have the same issue using the fine-tune method. I changed the learning rate twice (0.0005 and even 0.0001) but still after few ten's of thousand of iterations it prints "no detections" and I confirmed it after shutting the training down and try using "retinanet infer"...

I'm wondering if my objects aren't too small, I'm trying to detect aphids on plants (see image linked, [https://fromsmash.com/5_svMDKrmh-ct](url)) and I have a small dataset of 2840 images for training and 720 for validation. 
So do you think I should do more augmentations because Retinanet can handle that size of object of should move to segmentation algorithm because this is too small ? 

Thank you

**Comment by james-nvidia** [comment]
Please see https://github.com/NVIDIA/retinanet-examples/issues/158 where I opened you question as a new issue.

**Comment by nakul3112** [comment]
Hi, I am training using Nvidia odtk and backbone resnet50fpn, but getting focal loss as greater than 1. 
1)  Could someone give intuition about the `focal loss`. 
2) Also, I am not getting the reason why it shows `No detections` after some iterations
